packages/ec-push-cli/src/main.rs

```diff
@@ -121,7 +124,7 @@ fn main() -> Result<()> {
 
     let individual_generator = ec::IndividualGenerator {
         genome_generator: plushy_generator,
-        scorer,
+        scorer: FnScorer(scorer),
```

Comment:
Same thing here

Reply:
And the same problem with `wrap()` here as well.